.expeditor/config.yml

```diff
@@ -96,7 +96,11 @@ subscriptions:
       - built_in:rollover_changelog
       - bash:.expeditor/promote_harts_and_containers.sh:
           post_commit: true
+      - bash:.expeditor/publish-release-notes.sh:
+          post_commit: true
```

Comment:
Is every step in the expeditor config executed?
Or would this fail if the previous step failed?

Reply:
I believe when the first step fails the whole actionset fails. Is this correct @tduffield ?

Reply:
Ideally we would want to release a hab build every time we release an omnibus builds, but if that cannot happen due to some reasonand we absolutely need to make an omnibus release, we would have to comment out the section to promote_harts_and_containers in the config.yml and then the release would go through as expected. Is that correct?

Reply:
@schisamo is correct. If an action fails, the action set stops. There is a way to get around this using the `always_run` parameter. 

From a Release Engineering perspective, if we cannot release the Habitat Builds we should not be releasing any builds. They are all a single Release Artifact. However, you are correct @PrajaktaPurohit that in this scenario you would need to comment out the `promote_harts_and_containers` script, effectively removing Habitat from the release pipeline. 

I would strongly caution against treating the Chef Habitat packages as second class artifact. If a fix is absolutely needed for an omnibus release, chances are that it is absolutely needed for the Chef Habitat release as well.